CreateRolePolicyAsync method doesn't send the PolicyDecisionLogic field of RolePolicy or deosn't map to the right feild

```
var policy = new RolePolicy
        {
            DecisionStrategy = DecisionStrategy.Affirmative,
            Type = PolicyType.Role, // This is the only type of policy we can create with Keycloak.Net.Core
            Logic = PolicyDecisionLogic.Negative,
            Name = policyName,
            Description = $"Policy that grants access if user has the '{dto.RoleName}' role.",
            RoleConfigs = [new RoleConfig() { Id = role.Id, Required = true}],
        };
        var createdPolicy = await _keycloak.CreateRolePolicyAsync(_realm, client.Id, policy);
```

Caused by: com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `org.keycloak.representations.idm.authorization.Logic` from String "Negative": not one of the values accepted for Enum class: [POSITIVE, NEGATIVE] error on the keycloak server

--- a/src/Keycloak.Net.Core/Models/AuthorizationPermissions/AuthorizationPermission.cs
+++ b/src/Keycloak.Net.Core/Models/AuthorizationPermissions/AuthorizationPermission.cs
@@ -1,4 +1,5 @@
-﻿using Keycloak.Net.Common.Converters;
+﻿using System.Runtime.Serialization;
+using Keycloak.Net.Common.Converters;
 
 namespace Keycloak.Net.Models.AuthorizationPermissions;
 
@@ -16,10 +17,8 @@ public class AuthorizationPermission
 	[JsonConverter(typeof(JsonStringEnumConverter))]
 	public AuthorizationPermissionType Type { get; set; }
 
-	[JsonConverter(typeof(JsonStringEnumConverter))]
 	public PolicyDecisionLogic Logic { get; set; }
 
-	[JsonConverter(typeof(JsonStringEnumConverter))]
 	public DecisionStrategy DecisionStrategy { get; set; }
 
 	[JsonPropertyName("resourceType")]
@@ -35,9 +34,14 @@ public class AuthorizationPermission
 	public IEnumerable<string> PolicyIds { get; set; }
 }
 
+[JsonConverter(typeof(EnumMemberJsonStringEnumConverter<PolicyDecisionLogic>))]
 public enum PolicyDecisionLogic
 {
+
+	[EnumMember(Value = "POSITIVE")]
 	Positive,
+
+	[EnumMember(Value = "NEGATIVE")]
 	Negative
 }
 
@@ -47,9 +51,15 @@ public enum AuthorizationPermissionType
 	Resource
 }
 
+[JsonConverter(typeof(EnumMemberJsonStringEnumConverter<DecisionStrategy>))]
 public enum DecisionStrategy
 {
+	[EnumMember(Value = "UNANIMOUS")]
 	Unanimous,
+
+	[EnumMember(Value = "AFFIRMATIVE")]
 	Affirmative,
+
+	[EnumMember(Value = "CONSENSUS")]
 	Consensus
 }

--- a/src/Keycloak.Net.Core/Models/Clients/RolePolicy.cs
+++ b/src/Keycloak.Net.Core/Models/Clients/RolePolicy.cs
@@ -16,10 +16,8 @@ public class Policy
 	[JsonConverter(typeof(JsonStringEnumConverter))]
 	public PolicyType Type { get; set; }
 
-	[JsonConverter(typeof(JsonStringEnumConverter))]
 	public PolicyDecisionLogic Logic { get; set; }
 
-	[JsonConverter(typeof(JsonStringEnumConverter))]
 	public DecisionStrategy DecisionStrategy { get; set; }
 
 	[JsonPropertyName("config")]
@@ -40,10 +38,8 @@ public class RolePolicy
 	[JsonConverter(typeof(JsonStringEnumConverter))]
 	public PolicyType Type { get; set; } = PolicyType.Role;
 
-	[JsonConverter(typeof(JsonStringEnumConverter))]
 	public PolicyDecisionLogic Logic { get; set; }
 
-	[JsonConverter(typeof(JsonStringEnumConverter))]
 	public DecisionStrategy DecisionStrategy { get; set; }
 
 	[JsonPropertyName("roles")]
@@ -73,10 +69,8 @@ public class UserPolicy
 	[JsonConverter(typeof(JsonStringEnumConverter))]
 	public PolicyType Type { get; set; } = PolicyType.User;
 
-	[JsonConverter(typeof(JsonStringEnumConverter))]
 	public PolicyDecisionLogic Logic { get; set; }
 
-	[JsonConverter(typeof(JsonStringEnumConverter))]
 	public DecisionStrategy DecisionStrategy { get; set; }
 
 	[JsonPropertyName("users")]
@@ -97,10 +91,8 @@ public class GroupPolicy
 	[JsonConverter(typeof(JsonStringEnumConverter))]
 	public PolicyType Type { get; set; } = PolicyType.Group;
 
-	[JsonConverter(typeof(JsonStringEnumConverter))]
 	public PolicyDecisionLogic Logic { get; set; }
 
-	[JsonConverter(typeof(JsonStringEnumConverter))]
 	public DecisionStrategy DecisionStrategy { get; set; }
 
 	[JsonPropertyName("groups")]

